app/build.gradle

```diff
@@ -99,6 +99,11 @@ dependencies {
     testImplementation "org.mockito.kotlin:mockito-kotlin:4.1.0"
     androidTestImplementation 'org.mockito:mockito-android:5.2.0'
 
+    // Googlemaps dependencies
+    implementation 'com.google.android.gms:play-services-maps:18.1.0'
+    implementation 'com.google.android.gms:play-services-location:21.0.1'
+    implementation 'com.google.firebase:firebase-auth-ktx:21.1.0'
```

Comment:
Thank you for pointing this. I've removed :

`implementation 'com.google.android.gms:play-services-maps:18.1.0'`
`implementation 'com.google.firebase:firebase-auth-ktx:21.1.0'`

since we already import theme somewhere else (and I don't think that the googlemaps API needs the firebase dependency).

But regarding the versions, the documentation says that 18.1.0 and 21.0.1 are the latest :

https://developers.google.com/android/guides/setup?hl=fr
https://mvnrepository.com/artifact/com.google.android.gms/play-services-maps?repo=google

Please let me know if you were talking about something else.